Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -60,21 +57,27 @@ const ActivityIndicator = React.createClass({
       animating: true,
       color: Platform.OS === 'ios' ? GRAY : undefined,
       hidesWhenStopped: true,
-      size: 'small',
     };
   },
 
   render() {
     const {onLayout, style, ...props} = this.props;
     let sizeStyle;
-    switch (props.size) {
-      case 'small':
-        sizeStyle = styles.sizeSmall;
-        break;
-      case 'large':
-        sizeStyle = styles.sizeLarge;
-        break;
+    if (isNaN(props.size)) {
+      switch (props.size) {
+        case null:
```

Comment:
@satya164 If I remember correctly, it's related to how flow keeps complaining. That's probably messed my head a bit on this as well as on the custom prop checker. I'll revert it, and see how it goes.